src/js/component/team/TeamList.js

```diff
@@ -0,0 +1,39 @@
+import { Action } from '../../action/Action.js';
+import { $ } from '../../util/domSelection.js';
+export class TeamList {
+  constructor() {
+    this.$teamListContainer = $('div.team-list-container');
+
+    const $refreshTeamButton = $('#refresh-team-list');
+    $refreshTeamButton.addEventListener('click', () => {
+      Action.getTeams();
+    });
```

Comment:
```suggestion
$refreshTeamButton.addEventListener('click', Action.getTeams());
```
이렇게 간단히 작성하실수도 있습니다! ㅎㅎ